traffic-control-openshift/README.md

```diff
@@ -1,25 +1,61 @@
 # Traffic Control on OpenShift
 This demo demonstrates how to use traffic control to test applications running in [OpenShift](https://www.openshift.com/). It uses OpenShift version 3 to run the Kubernetes cluster, [tcd](https://github.com/kinvolk/tcd) to do traffic control on the Kubernetes nodes and [Weave Scope](https://github.com/weaveworks/scope) to monitor, visualize and interact with the Kubernetes cluster.
 
-## What do you need to start?
+To find more information about traffic control and this demo check our [blog post](https://kinvolk.io/blog/2016/05/testing-web-services-with-traffic-control-on-kubernetes/).
+
+## Prepare for the demo
+### What do you need to start?
 - Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You will need to reboot the VM after the first `vagrant up` in order to load the right kernel version and to load the modules needed by tcd.
 - Get the [demo](https://github.com/kinvolk/demo) repository
 
-You can reach the OpenShift dashboard by pointing your browser to [https://10.2.2.2:8443](https://10.2.2.2:8443). Use user 'admin' and password 'admin' to login.
+### Get the VM
+```
+git clone https://github.com/kinvolk/openshift-evangelists-vagrant-origin.git
+cd openshift-evangelists-vagrant-origin
+vagrant up
+```
+The building process could take several minutes (15 minutes) depending of your machines and the output should look like this:
```

Comment:
you could add why it takes time: it is installing additional packages on top of a Fedora base image, and then fetching the last openshift sources and then build it.